site/documentation/content/api/tenant/index.md

```diff
@@ -170,17 +171,33 @@ The table below provides an overview of the members defined for the *tracing* JS
 
 ### Trusted CA Format
 
-The table below provides an overview of the members defined for the *trusted-ca* JSON object:
+If the *trusted-ca* property is provided, then it MUST contain a non-empty array of JSON objects, were each JSON object represents a single trusted CA. The table below provides an overview of the members of a JSON object representing a trusted CA:
 
 | Name                     | Mandatory  | Type          | Default Value | Description |
 | :------------------------| :--------: | :------------ | :------------ | :---------- |
-| *subject-dn*             | *yes*      | *string*      |               | The subject DN of the trusted root certificate in the format defined by [RFC 2253](https://www.ietf.org/rfc/rfc2253.txt). |
-| *cert*                   | *no*       | *string*      |               | The Base64 encoded binary DER encoding of the trusted root X.509 certificate. |
-| *public-key*             | *no*       | *string*      |               | The Base64 encoded binary DER encoding of the trusted root certificate's public key. |
+| *subject-dn*             | *yes*      | *string*      |               | The subject DN of the trusted root certificate in the format defined by [RFC 2253](https://www.ietf.org/rfc/rfc2253.txt). Trusted CAs of the same tenant MAY share the same subject DN (e.g. allowing for the definition of overlapping validity periods). However, trusted CAs of different tenants MUST NOT share the same subject DN in order to allow for the look up of a tenant by the subject DN of one of its trusted CAs. |
+| *public-key*             | *yes*       | *string*     |               | The Base64 encoded binary DER encoding of the trusted root certificate's public key. |
 | *algorithm*              | *no*       | *string*      | `RSA`        | The name of the public key algorithm. Supported values are `RSA` and `EC`. This property is ignored if the *cert* property is used to store a certificate. |
 
-* The *subject-dn* MUST be unique among all registered tenants.
-* Either the *cert* or the *public-key* MUST be set.
+**Examples**
+
+Below is an example for a payload of the response to a *get* request for tenant `TEST_TENANT`. The tenant is configured with two valid trusted certificate authorities having the same *public-key* and *subject-dn* but with overlapping validity periods.
```

Comment:
they obviously do not have the same public key and it is hard to tell whether they have overlapping validity periods, but they are definitely valid right now

Reply:
the example JSON evolved overtime but not the surrounding text :-)